cmd/lnd/main.go

```diff
@@ -10,9 +10,16 @@ import (
 )
 
 func main() {
+	// Hook interceptor for os signals.
+	signal, err := signal.Intercept()
```

Comment:
same re: trying to avoid colliding with the package name

Reply:
Fixed.